### PR TITLE
docs: timer-picker remove clearIcon from document

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- https://github.com/ant-design/ant-design/discussions/43582

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | timer-picker remove clearIcon from document |
| 🇨🇳 Chinese | 从文档中删除时间选择器的清除按钮  |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4c77894</samp>

Removed `clearIcon` prop from `TimePicker` component API documentation in both English and Chinese. This is part of a pull request that updates the `TimePicker` component to use the new `rc-picker` library.

### <samp>🤖 Generated by Copilot at 4c77894</samp>

*  Remove `clearIcon` prop from `TimePicker` component API documentation in both English and Chinese, as it is not supported by the new `rc-picker` library that the component uses (`[link](https://github.com/ant-design/ant-design/pull/44977/files?diff=unified&w=0#diff-93e8155915622d7eec5ef5f00cc60b8a700baed676ab713aca37c6281a82c310L58)`, `[link](https://github.com/ant-design/ant-design/pull/44977/files?diff=unified&w=0#diff-2dda2de66ee5e0215c478323e414235aa4613c4d93674510732ef1d043508061L58)`)

### CODE REVIEW

File: components/time-picker/index.en-US.md [hunk 1]
```diff
@@ -55,7 +55,6 @@ dayjs.extend(customParseFormat)
 | cellRender | Custom rendering function for picker cells | (current: number, info: { originNode: React.ReactElement, today: dayjs, range?: 'start' \| 'end', subType: 'hour' \| 'minute' \| 'second' \| 'meridiem' }) => React.ReactNode | - | 5.4.0 |
 | changeOnBlur | Trigger `change` when blur. e.g. datetime picker no need click confirm button | boolean | false | 5.5.0 |
 | className | The className of picker | string | - |  |
-| clearIcon | The custom clear icon | ReactNode | - |  |
```
Comment (afc163) [inline components/time-picker/index.en-US.md:58]: Select 里还有。

**Comment by yoyo837** [comment]
是否关联一下之前的issue或者PR?

File: components/time-picker/index.en-US.md (hunk 1 above)
Comment (kiner-tang) [reply components/time-picker/index.en-US.md:58]: 应该也是文档漏删了

Comment (kiner-tang) [reply components/time-picker/index.en-US.md:58]: 我晚上全部捋一下，看看还有没有漏的

**Comment by kiner-tang** [comment]
> 是否关联一下之前的issue或者PR?

把 RFC 链接加上了

**Review by afc163 — APPROVED** [review]

**Comment by crazyair** [comment]
`从文档中删除时间选择器的清楚按钮` 清除

**Comment by kiner-tang** [comment]
> `从文档中删除时间选择器的清楚按钮` 清除

更新了

**Pull request merged**